package.json

```diff
@@ -0,0 +1,68 @@
+{
+    "license": "MIT",
+    "version": "1.0.0",
+    "description": "Active Choices Plugin UI",
+    "name": "uno-choice",
+    "private": true,
+    "scripts": {
+        "test": "jest",
+        "dev": "webpack --config webpack.config.js",
+        "prod": "webpack --config webpack.config.js --mode=production",
+        "build": "yarn prod",
+        "mvntest": "yarn test",
+        "mvnbuild": "yarn build"
+    },
+    "devDependencies": {
+        "@babel/cli": "^7.22.5",
+        "@babel/core": "7.22.5",
+        "@babel/preset-env": "^7.22.5",
+        "@babel/preset-typescript": "^7.22.5",
+        "@babel/preset-flow": "^7.22.5",
+        "eslint": "8.43.0",
+        "eslint-config-prettier": "8.8.0",
+        "jest": "^29.5.0",
+        "jest-environment-jsdom": "^29.5.0",
+        "jest-junit": "^16.0.0",
+        "jsdom": "^22.1.0",
+        "prettier": "2.8.8",
+        "ts-loader": "^9.4.3",
+        "typescript": "^5.1.3",
+        "webpack": "5.87.0",
+        "webpack-cli": "5.1.4",
+        "jquery": "^3.7.0",
+        "@types/jquery": "^3.5.16"
+    },
+    "dependencies": {
```

Comment:
It will be nice if someday Jenkins move to having a separate UI layer, so that we can declare dependencies to plug-ins and other libs here (like the JQuery one). Although that's probably hard with stapler and all the plugin tooling.